Example/GiniVision.xcodeproj/xcshareddata/xcschemes/GiniVision-Example.xcscheme

```diff
@@ -42,7 +42,7 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "FA4C1717ACBA7DAFF610065C12D00D2C"
+               BlueprintIdentifier = "0F25960EE62BCB781A7B936531602322"
```

Comment:
What is this?

Reply:
Weird Xcode stuff. I imagine that after trying several signing configurations, it changed.